src/sentry/nodestore/bigtable/backend.py

```diff
@@ -172,7 +172,7 @@ def set(self, id, data, ttl=None):
         self._set_cache_item(id, data)
 
     def encode_row(self, id, data, ttl=None):
-        data = json_dumps(data)
+        data = json_dumps(data).encode("utf-8")
```

Comment:
Just want to confirm you tested on both py2 and py3?

Reply:
@EvanPurkhiser verified this just now, everything looks good.

Reply:
Verified this in py2, tests pass and I checked with unicode, works no problem.